regnety/profiler.py

```diff
@@ -0,0 +1,64 @@
+import os
+os.environ['TF_CPP_MIN_LOG_LEVEL'] = '3' 
+import tensorflow as tf
+import pandas as pd
+import numpy as np
+import time
+from PIL import Image
+#%matplotlib inline
+import matplotlib.pyplot as plt
+
+
+from dataset import imagenet
+
+
+
+tf.keras.backend.clear_session()
+
+imgnet = imagenet.ImageNet(
+    tf.io.gfile.glob('gs://adityakane-imagenette-tfrecs/*.tfrecord'),
```

Comment:
This path should come from a config script.

Reply:
Actually profiler.py is a dummy script, just like main.py. I'll delete it when it seems unnecessary. Is that okay?

Reply:
Fair enough.